src/inmanta/parser/plyInmantaParser.py

```diff
@@ -692,6 +692,33 @@ def p_boolean_expression_is_defined_short(p: YaccProduction) -> None:
     attach_lnr(p)
 
 
+def p_boolean_expression_is_defined_map_lookup_0(p: YaccProduction) -> None:
+    """boolean_expression : is_defined_expr IS DEFINED"""
+    p[0] = p[1]
+
+
+def p_is_defined_expr_is_defined_map_lookup_1(p: YaccProduction) -> None:
```

Comment:
This took me some time to understand, but the parser generator completely collapses `is_defined_expr` and `map_lookup` until at some point it does

```    IS              reduce using rule 116 (is_defined_expr -> var_ref [ operand ] .)```

Reply:
Is this a concern?